components/dashboard/dashboard.go

```diff
@@ -68,12 +69,31 @@ func (d *Dashboard) ReconcileComponent(cli client.Client, owner metav1.Object, d
 			if err != nil {
 				return err
 			}
+
+			// Deploy CRDs for odh-dashboard
```

Comment:
should this block put out of the if-block? 
if we do not care which platform as long as it is enabled component

Reply:
another thing, in deploy.go https://github.com/opendatahub-io/opendatahub-operator/blob/f43f09b30be92fbc8984dce20a00a7fe79e8f3ce/pkg/deploy/deploy.go#L233 should we update to 
```suggestion
if found.GetKind() == "CustomResourceDefinition" && compFound != "odh-dashboard" {
			// Deploy CRDs for odh-dashboard
				return nil
			}
```

Reply:
Also for the deploy, I think we should not be cleaning up the CRDs even for dashboard...right ?

Reply:
i am not really sure about the CRD  part. 
if UI team is keeping update CRD schema, when an old CRD has been left in the cluster, will the new one be applied to cluster (assume both with same api version v1alpha1) unless they are always backwards compatible, e.g no need conversion.

Reply:
@etirelli how you feel about these dashboard CRDs, should remove them if dashboard is disabled or just keep them?

Reply:
ok, we can keep CRD in the cluster, as long as UI team make sure any new changes are backwards compatible e.g when add the new CRD to overwrite the old CRD wont be any conflict